misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/BTestMain.java

```diff
@@ -95,9 +99,18 @@ public static void main(String[] args) throws IOException {
             }
             out.println();
 
-            Path testSuiteCachePath = testCache.resolve(TesterinaConstants.TESTERINA_TEST_SUITE);
+            BufferedReader br;
+            if (Files.notExists(testSuiteJsonPath)) {
+                InputStream is = BTestMain.class.getResourceAsStream(TesterinaConstants.PATH_SEPARATOR
+                        + testSuiteJsonPath);
+                assert is != null;
+                br = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8));
+            }
+            else {
+                br = Files.newBufferedReader(testSuiteJsonPath, StandardCharsets.UTF_8);
+            }
 
-            try (BufferedReader br = Files.newBufferedReader(testSuiteCachePath, StandardCharsets.UTF_8)) {
+            try (br) {
```

Comment:
LEt's use try with resources

Reply:
Done in [b26b6e5](https://github.com/ballerina-platform/ballerina-lang/pull/42183/commits/b26b6e545aebed16500e2530f29f5ba5253e6502) commit